automatic/selenium.powershell/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,51 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$ModuleName = "Selenium"
+$ModuleVersion = $env:ChocolateyPackageVersion
+
+$SourcePath = Join-Path $toolsDir "$($ModuleName).zip"
+$SavedPaths = Join-Path $ToolsDir 'installedpaths'
+
+$DestinationPath = switch ((Get-PackageParameters).Keys) {
+  "Core" {
+    Join-Path $env:ProgramFiles "PowerShell\Modules"
+  }
+  "Desktop" {
+    Join-Path $env:ProgramFiles "WindowsPowerShell\Modules"
+  }
+}
+
+# By default, install for Windows PowerShell
+if (-not $DestinationPath) {
+  $DestinationPath = Join-Path $env:ProgramFiles "WindowsPowerShell\Modules"
+}
+
+$UnzipArgs = @{
+  FileFullPath = $SourcePath
+  PackageName  = $env:ChocolateyPackageName
+}
+
+$InstalledPaths = foreach ($Path in $DestinationPath) {
```

Comment:
Any reason why you are using a `foreach` loop, instead of doing piping and the `ForEach-Object` cmdlet?

Reply:
Mostly because I picked it up from @JPRuskin's previous work. I've updated it to using `ForEach-Object` to be more consistent with how PowerShell would normally write the assignment.